.github/workflows/appengine-deploy.yml

```diff
@@ -52,3 +56,15 @@ jobs:
       - id: test
         name: Test the deloyed URL
         run: curl --fail "${{ steps.deploy.outputs.url }}"
+        
+      - name: Happo Screenshot Testing
+        working-directory: web
+        run: npm run happo-ci-github-actions
+        env:
+          PREVIOUS_SHA: 490c6fa004074f6cf23649f5336f1e9004f4574d
```

Comment:
Can you put the `490c6fa004074f6cf23649f5336f1e9004f4574d` in a `GOLDEN_HASH` in `env` section starting on line 9? A comment that explains the meaning and when to update it would also help.